ReactNativeTraining/src/app/screens/Home/screens/Chats/layout.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react';
+
+import ContactList from '../../../components/ContactList';
+
+class Chats extends React.Component {
+  isAMessageBetween = (userId, contactId, message) =>
+    (message.senderId === userId && message.receiverId === contactId) ||
+    (message.receiverId === userId && message.senderId === contactId);
+
+  lastMessage = (userId, contactId) =>
```

Comment:
Put this both functions in container component. Check if u can use `Reselect` fuctions here.